packages/cardano-services/src/Program/loadHttpServer.ts

```diff
@@ -53,6 +54,19 @@ export const loadHttpServer = async (args: ProgramArgs): Promise<HttpServer> =>
           })
         );
         break;
+      case ServiceNames.ChainHistory:
+        if (args.options?.dbConnectionString === undefined)
+          throw new MissingProgramOption(ServiceNames.ChainHistory, ProgramOptionDescriptions.DbConnection);
+        services.push(
+          await ChainHistoryHttpService.create({
+            chainHistoryProvider: new DbSyncChainHistoryProvider(
+              new Pool({ connectionString: args.options.dbConnectionString }),
```

Comment:
Should we hoist this before the switch to share the instance across services?

Reply:
Done in 70dd1e23cbb08257e051305e215dc03c001d0d99